.github/CODEOWNERS

```diff
@@ -431,6 +431,7 @@
 /pkg/collector/corechecks/network-devices/         @DataDog/ndm-integrations
 /pkg/collector/corechecks/orchestrator/   @DataDog/container-app
 /pkg/collector/corechecks/net/          @DataDog/agent-runtimes
+/pkg/collector/corechecks/net/wlan/     @DataDog/windows-agent
```

Comment:
I believe wlan is owned by agent-runtimes, windows-agent should probably just own the windows implementation ?

Reply:
No, the change was deliberate and not incidental. Will discuss OOTB.